npm_and_yarn/lib/dependabot/npm_and_yarn/package_manager.rb

```diff
@@ -96,7 +96,11 @@ def guessed_version(name)
         lockfile = @lockfiles[name.to_sym]
         return unless lockfile
 
-        Helpers.send(:"#{name}_version_numeric", lockfile)
+        version = Helpers.send(:"#{name}_version_numeric", lockfile)
```

Comment:
can we get a metric to datadog for this?

Reply:
Sorry branchname is changed, PR is closed automatically. New PR: https://github.com/dependabot/dependabot-core/pull/10757